src/test/java/com/commercetools/project/sync/CliRunnerTest.java

```diff
@@ -195,6 +196,32 @@ void run_WithSyncAsArgumentWithNoArgs_ShouldFailAndLogError() {
             });
   }
 
+  @Test
```

Comment:
The order of the functions looks weird, maybe put it both tests after `run_AsProductFullSync_ShouldBuildSyncerAndExecuteSync` not middle of product delta and product full sync test.

Reply:
https://github.com/commercetools/commercetools-project-sync/commit/546fdeb0831dccb3ee1d3823e23df2afbfa54b34